esp/platform/espcfg.cpp

```diff
@@ -400,18 +400,6 @@ CEspConfig::CEspConfig(IProperties* inputs, IPropertyTree* envpt, IPropertyTree*
         if (sdsSessionNeeded && !daliservers.isEmpty())
             initSDSSessionCleaner(isDetachedFromDali());
 
-#ifndef _CONTAINERIZED
-        // Copy the 'cost' tree as provided in environment.xml to the globalConfiguration IPropertyTree,
-        // so that it is available to the functions that calculate storage and workunit costs
-        // (Thor, hThor and eclagent are already using globalConfiguration, so this copying is not nessary anywhere else)
-        IPropertyTree * espConfigCost = m_envpt->queryPropTree("cost");
-        if (espConfigCost)
-        {
-            Owned <IPropertyTree> global = getGlobalConfig();
-            IPropertyTree * cost = global->hasProp("cost") ? global->queryPropTree("cost") : global->addPropTree("cost");
-            mergeConfiguration(*cost, *espConfigCost, nullptr, false);
-        }
-#endif
```

Comment:
Just curious. Did old code work before? Wondering why not using m_envpt->queryPropTree("global/cost") .